src/SFA.DAS.AANHub.Api/Controllers/EmployersController.cs

```diff
@@ -31,12 +32,12 @@ public EmployersController(ILogger<EmployersController> logger, IMediator mediat
         [Produces("application/json")]
         [ProducesResponseType(typeof(string), StatusCodes.Status400BadRequest)]
         [ProducesResponseType(typeof(string), StatusCodes.Status201Created)]
-        public async Task<IActionResult> CreateEmployer([FromHeader(Name = Constants.PostRequestHeaders.RequestedByUserHeader)] [Required] Guid? userId,
+        public async Task<IActionResult> CreateEmployer([FromHeader(Name = Constants.PostRequestHeaders.RequestedByUserHeader)] [Required] Guid userId,
             CreateEmployerModel request)
         {
             _logger.LogInformation("AAN Hub API: Received command to add employer by accountId: {accountId} and UserId: {userId}",
                 request.AccountId,
-                request.UserId);
```

Comment:
It is worth to include the requesting user id in the log

Reply:
Added requesting user id into the log